src/main/webapp/app/exam/participate/exercises/quiz/quiz-exam-submission.component.html

```diff
@@ -6,27 +6,27 @@
                     <span
                         *ngIf="question.type === DRAG_AND_DROP"
                         class="btn btn-light btn-circle stepbutton stepwizardquiz-circle draganddropcolor-question"
-                        (click)="navigateToQuestion(question.id)"
-                        [ngbTooltip]="dragAndDropMappings[question.id]?.length ? tooltipExplanationTranslate : tooltipNotExplanationTranslate"
-                        [ngClass]="!!dragAndDropMappings.get(question.id)?.length ? 'changed-question' : ''"
+                        (click)="navigateToQuestion(question.id!)"
+                        [ngbTooltip]="dragAndDropMappings[question.id!]?.length ? tooltipExplanationTranslate : tooltipNotExplanationTranslate"
+                        [ngClass]="!!dragAndDropMappings.get(question.id!)?.length ? 'changed-question' : ''"
```

Comment:
A check to not show it, if the question does not have an id then it might cause runtime errors. If the question definitely has id here then this should work.

Reply:
@iozbeyli You mean I should filter out all questions without id before they get displayed?

Reply:
No in this case this is okay since ids should be there